packages/py/pyenchant/package.yml

```diff
@@ -1,8 +1,9 @@
 name       : pyenchant
-version    : 2.0.0
-release    : 13
+version    : 3.2.2
+release    : 14
 source     :
-    - https://files.pythonhosted.org/packages/source/p/pyenchant/pyenchant-2.0.0.tar.gz : fc31cda72ace001da8fe5d42f11c26e514a91fa8c70468739216ddd8de64e2a0
+    - https://files.pythonhosted.org/packages/b1/a3/86763b6350727ca81c8fcc5bb5bccee416e902e0085dc7a902c81233717e/pyenchant-3.2.2.tar.gz : 1cf830c6614362a78aab78d50eaf7c6c93831369c52e1bb64ffae1df0341e637  
```

Comment:
Please use this source instead: https://files.pythonhosted.org/packages/source/p/pyenchant/pyenchant-3.2.2.tar.gz